viewing_party/party.py

```diff
@@ -1,23 +1,126 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dictionary = {}
+    if title and genre and rating:
+        movie_dictionary["title"] = title
+        movie_dictionary["genre"] = genre
+        movie_dictionary["rating"] = rating
+    else:
+        return None
+    
+    return movie_dictionary
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    copy_user_data = user_data
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            copy_user_data["watched"].append(movie)
+            copy_user_data["watchlist"].remove(movie)
+        user_data = copy_user_data

```

Comment:
Since line 25 was `copy_user_data = user_data` which meant that the 2 variables are pointing to the same places in memory then line 30 isn't changing anything. We're now saying user_data points to copy_user_data, which means they're both pointing to the same place in memory containing the user_data list. So we could remove line 25 and 30 without affecting how your function works